spec/facility_spec.rb

```diff
@@ -24,3 +27,61 @@
     end
   end
 end
+
+require 'spec_helper'
+
+RSpec.describe Facility do
+  before(:each) do
+    @facility_1 = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+    @facility_2 = Facility.new({name: 'Ashland DMV Office', address: '600 Tolman Creek Rd Ashland OR 97520', phone: '541-776-6092' })
+    @cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+    @bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+    @camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} ) 
+  end
+  
+  describe '#initial service registration' do
+    it 'can add vehicle registration' do
+      
+      @facility_1.add_service('Vehicle Registration')
```

Comment:
This is good, and we'd want to start this test checking that the `@facility_1.services` starts off as an empty array. Then we do the rest of the test as your wrote to ensure it changes like we'd want.